docker-compose.yml

```diff
@@ -55,3 +57,16 @@ services:
         retries: 6
     ports:
       - "1433:1433"
+
+  sql-data:
+    image: "mcr.microsoft.com/mssql/server"
+    environment:
+      SA_PASSWORD: ${SAPASSWORD}
+      ACCEPT_EULA: "Y"
+    healthcheck:
+        test: ["CMD", "/opt/mssql-tools/bin/sqlcmd","-S sql-data,1434 -U sa -P ${SAPASSWORD} -Q 'SELECT * FROM INFORMATION_SCHEMA.TABLES'"]
+        interval: 10s
+        timeout: 10s
+        retries: 6
+    ports:
+      - "1434:1433"
```

Comment:
I believe we only need to specify ports if we want to service to be exposed externally. I'd query if either SQL server need to be exposed externally. If not, try not exposing ports at all, they should both be accessible on 1433...

Doesn't really matter I guess as using compose for dev, but aware the port numbers are getting messier.

Reply:
QA are connecting to sql on their host machines. Useful for now.

Yeh, the ports will get messier once we get more micro services.